Signal/src/view controllers/AdvancedSettingsTableViewController.m

```diff
@@ -18,12 +18,12 @@
 
 @interface AdvancedSettingsTableViewController ()
 
-@property (nonatomic) UITableViewCell *enableCallKitCell;
+@property (nonatomic) UITableViewCell *enableCallKitPrivacyCell;
 @property (nonatomic) UITableViewCell *enableLogCell;
 @property (nonatomic) UITableViewCell *submitLogCell;
 @property (nonatomic) UITableViewCell *registerPushCell;
 
-@property (nonatomic) UISwitch *enableCallKitSwitch;
```

Comment:
Done. Unfortunately, I had to do this work twice, because I did half of it at the airport and somehow lost the commit/work.